spec/hyperlink-spec.coffee

```diff
@@ -29,6 +29,9 @@ describe 'Hyperlink grammar', ->
     {tokens} = plainGrammar.tokenizeLine 'https://github.com/atom/brightray_example'
     expect(tokens[0]).toEqual value: 'https://github.com/atom/brightray_example', scopes: ['text.plain.null-grammar', 'markup.underline.link.https.hyperlink']
 
+    {tokens} = plainGrammar.tokenizeLine 'https://sv.wikipedia.org/wiki/Mañana'
+    expect(tokens[0]).toEqual {value: 'https://sv.wikipedia.org/wiki/Mañana', scopes: ['text.plain.null-grammar', 'markup.underline.link.https.hyperlink']}, 'allows unicode characters'
```

Comment:
The `'allows unicode characters'` part shouldn't be there.  If you want to clarify what this test is doing, you can create a new `it` block.